### PR TITLE
add verify command

### PR DESCRIPTION
**Pull request by RitikJaiswal75**
This PR adds a verify command to our bot which sends a direct message to the user.

Currently, the message is just hello the actual link with the token will come up in the consecutive PRs.

To test out the PR in action please join my server: https://discord.gg/tRQrgm7D
and send the `/hello` and `/verify` commands.

### CODE REVIEW

File: src/utils/sendDiscordDm.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      }),
+    }
+  ).then((res) => {
+    return res.json();
+  });
+
+  const channelId = createDmChannel.id;
+
+  // "/channels/{channel.id}/messages" is used to send a message to the specified channel
+  await fetch(`${DISCORD_BASE_URL}/channels/${channelId}/messages`, {
```
Comment (heyrandhir) [inline src/utils/sendDiscordDm.ts:38]: we should wrap this up in a `try - catch` block

**Review by heyrandhir — CHANGES_REQUESTED** [review]

File: src/utils/sendDiscordDm.ts (hunk 1 above)
Comment (RitikJaiswal75) [reply src/utils/sendDiscordDm.ts]: thanks for the comment. I added it.

File: src/utils/sendDiscordDm.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+/**
+ *
+ * @param userId {number}: user id of the person using the slash command received from the interaction
+ * @param env {env}: contains environment variables
+ */
+
+export const sendDiscordDm = async (userId: number, env: env) => {
+  // "/users/@me/channels" is an endpoint provided by discord to create a dm channel
+  const createDmChannel: createDmChannel = await fetch(
```
[the remark this answers is not in the document]
Comment (RitikJaiswal75) [reply src/utils/sendDiscordDm.ts]: thanks for the comment. I added it.

**Review by prakashchoudhary07 — APPROVED** [review]
LGTM 👍

**Review by heyrandhir — COMMENTED** [review]
Lgtm

**Review by heyrandhir — APPROVED** [review]

**Review by samyakshah3008 — COMMENTED** [review]
Great work, Ritik! 
Looks good to merge!

**Pull request merged**